### PR TITLE
Improve readability of release notes

### PR DESCRIPTION
**Pull request by abitrolly**
https://screenshots.firefox.com/u9k8X2UfLGsxaPoZ/bodhi.fedoraproject.org

I am not sure if Sphinx will pick up this stylesheet, but there is an empty custom.css in web site sources, so it may work.

### CODE REVIEW

**Review by bowlofeggs — APPROVED** [review]
It seems to work OK with sphinx-build, so I expect it'll work in production the same way. Thanks!

**Comment by bowlofeggs** [comment]
@abitrolly Can you edit your commit message to sign the Developer Certificate of Origin, as described here?

https://bodhi.fedoraproject.org/docs/developer/index.html#contribution-guidelines

**Review by bowlofeggs — CHANGES_REQUESTED** [review]
@abitrolly Can you edit your commit message to sign the Developer Certificate of Origin, as described here?

https://bodhi.fedoraproject.org/docs/developer/index.html#contribution-guidelines

**Comment by abitrolly** [comment]
I thought GitHub signs it automatically with Verified mark. Adjusted manually.

**Comment by bowlofeggs** [comment]
@abitrolly the verified mark means that the commit was gpg signed, but that's different than the DCO agreement. Thanks, looks good now!

**Review by bowlofeggs — APPROVED** [review]

**Comment by abitrolly** [comment]
How does that improve the workflow if developers will still have to go to the web interface to approve the PR? Instead of approving and waiting for merge and fail, it would be faster to do rebase and merge directly, no?

**Comment by bowlofeggs** [comment]
@abitrolly I'm not sure I understand your question. Can you rephrase?

**Comment by abitrolly** [comment]
The comment is about **mergify**.

**Comment by bowlofeggs** [comment]
@abitrolly well Mergify doesn't just do a rebase and merge, it also waits for CI to pass. There are times that a rebased patch fails CI when it passed previously (because another patch was merged), so this is intended to catch those sorts of issues. If I were to run the tests on every patch after rebasing and then merge, that would take a lot of my time.

**Comment by abitrolly** [comment]
Waiting for CI makes sense.
